NuGet.Config

```diff
@@ -16,6 +16,7 @@
     <add key="dotnet.myget.org dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/" />    
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
+    <add key="dotnet.myget.org cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
```

Comment:
Agreed.  Only the "cli" repo should depend on "cli-deps".

@Petermarcu - which package(s) did you need this for?